esp32_s3_lcd_ev_board/Kconfig

```diff
@@ -0,0 +1,121 @@
+
+        config BSP_LCD_RGB_DOUBLE_BUFFER
+            bool "Enable to use double frame buffers"
+            depends on BSP_LCD_SUB_BOARD_480_480 || BSP_LCD_SUB_BOARD_800_480
+            default "n"
+            help
+                Let RGB LCD driver create two frame size buffers. It's a necessary configuration to avoiding tearing effect.
+
+        config BSP_LCD_RGB_REFRESH_TASK_ENABLE
+            bool "Enable to refresh LCD manually"
+            depends on BSP_LCD_SUB_BOARD_480_480 || BSP_LCD_SUB_BOARD_800_480
+            default "n"
+            help
+                Refresh the LCD in a specific task instead of automatically refreshing. It's a necessary configuration to avoiding tearing effect.
+
+        config BSP_LCD_RGB_REFRESH_TASK_PERIOD
+            int "Minimum Period(ms) of LCD refreshing task"
+            depends on BSP_LCD_RGB_REFRESH_TASK_ENABLE
+            default 10
+            help
+               This configuration does not necessarily represent the actual refresh cycle of the RGB interface.
+
+        config BSP_LCD_RGB_REFRESH_TASK_PRIORITY
+            int "Priority of LCD refreshing task"
+            depends on BSP_LCD_RGB_REFRESH_TASK_ENABLE
+            default 2
+    endmenu
+
+    menu "Display"
+        config BSP_DISPLAY_LVGL_TASK_PRIORITY
+            int "LVGL task priority"
+            default 2
+            help
+                The Board Support Package will create a task that will periodically handle LVGL operation in lv_timer_handler().
+
+        config BSP_DISPLAY_LVGL_TICK
+            int "LVGL tick period"
+            default 5
+            range 1 100
+            help
+                Period of LVGL tick timer.
+
+        config BSP_DISPLAY_LVGL_AVOID_TEAR
+            bool "Avoid tearing effect"
+            depends on BSP_LCD_RGB_REFRESH_TASK_ENABLE && BSP_LCD_RGB_DOUBLE_BUFFER
+            default "n"
+            help
+                Avoid tearing effect through LVGL buffer mode and double frame buffers of RGB LCD. This feature is only available for RGB LCD.
+
+        choice BSP_DISPLAY_LVGL_MODE
+            depends on BSP_DISPLAY_LVGL_AVOID_TEAR
+            prompt "Select LVGL buffer mode"
+            default BSP_DISPLAY_LVGL_FULL_REFRESH
+            config BSP_DISPLAY_LVGL_FULL_REFRESH
+                bool "Full refresh"
+            config BSP_DISPLAY_LVGL_DIRECT_MODE
+                bool "Direct mode"
+        endchoice
+
+        choice BSP_DISPLAY_LVGL_BUF_CAPS
+            depends on !BSP_DISPLAY_LVGL_AVOID_TEAR
+            prompt "Select LVGL buffer memory capability"
+            default BSP_DISPLAY_LVGL_INTERNAL
+            config BSP_DISPLAY_LVGL_PSRAM
+                bool "PSRAM memory"
+            config BSP_DISPLAY_LVGL_INTERNAL
+                bool "Internal memory"
+        endchoice
+
+        config BSP_DISPLAY_LVGL_BUF_HEIGHT
+            depends on !BSP_DISPLAY_LVGL_AVOID_TEAR
+            int "LVGL buffer height"
+            default 10
```

Comment:
I tried this BSP with SquareLine - used their Thermostat example and it was really slow. I had to changed this to 100, for faster. I think, it can be more. Why the default is 10? Could it be defoult more (e.g. 100) ?

Reply:
I didn't think too much and just use the minimum size of the LVGL buffer. Sure, I can increase it to 100, PTAL [here](https://github.com/espressif/esp-bsp/pull/89/commits/927713d146b33c808aa07e634235995d44adb973#diff-918c0433b106ae9d861b42e18180e71a6a564c01719cbe2bcb38f2984a8dcf17L110-R110).

Reply:
Thank you.